mappings/net/minecraft/block/SmithingTableBlock.mapping

```diff
@@ -1 +1,2 @@
 CLASS net/minecraft/class_3717 net/minecraft/block/SmithingTableBlock
+	FIELD field_22511 SCREEN_TITLE Lnet/minecraft/class_2588;
```

Comment:
I would prefer `SCREEN_NAME`, or even use the good old `CONTAINER_NAME`

Reply:
this is like the title that's sent for the window to be opened on client https://wiki.vg/Protocol#Open_Window